app/Jobs/ProcessSubmission.php

```diff
@@ -126,7 +126,7 @@ public function handle()
             $this->requeueSubmissionFile($build->Id);
         }
 
-        if (config('cdash.backup_timeframe') === 0) {
+        if (config('cdash.backup_timeframe') == 0) {
```

Comment:
Any reason why you can't cast the config value to an int and then do strict comparison?

```suggestion
        if ((int) config('cdash.backup_timeframe') === 0) {
```